toy_app/app/views/microposts/show.html.erb

```diff
@@ -0,0 +1,15 @@
+<style>
+.title{
+  font-size: 18px; font-weight: bold;
+}
+</style>
```

Comment:
Also for this style, you should't place them in this file, you should move it to a css/scss file instead